### PR TITLE
added styles for typography

### PR DESCRIPTION
**Pull request by KarynaKovhan**
typography

### CODE REVIEW

**Review by akazyryna — COMMENTED** [review]
`<span>WHERE TO GO NEXT</span>`- it's not span
              `<span>` -it's also not span
                `<p><ul>Many Cuisenaire and Gattegno resources can be found online:</p>` - why you added ul and li tags to the p tag?
               `<p><li>New developments in arithmetic teaching in Britain: introducing the concept of ‘set’ 
                <a href="../html/index.html">helloworld.cc/2pYFgOw</a>.</li></p>`
              `<p><li>The common sense of teaching mathematics: 
              <a href="../html/index.html">helloworld.cc/2p9dyKO</a></li></p>`
             `<p><li>Gattegno Mathematics, Book 1: 
             <a href="../html/index.html">helloworld.cc/2qz7xZe</a>.</li></p>`
              `<p><li>Virtual Rods: <a href="../html/index.html">helloworld.cc/2pI8ctc</a>.</li></p>`
              `<p><li>Sample code for these exercises: <a href="../html/index.html">helloworld.cc/2pFyx9j</a>.</li></p>`
            ` </ul></span>`

**Comment by KarynaKovhan** [comment]
Ok
I added ul li tags in order to make red dots near the links. If it’s
inappropriate, I”ll change them

Вт, 3 мая 2022 г. в 14:38, Alina Kazyryna ***@***.***>:

**Review by akazyryna — COMMENTED** [review]
It's unreadable, please make nice indents for tags and text. Need to apply to the entire html doc

**Comment by akazyryna** [comment]
> I added ul li tags in order to make red dots near the links. If it’s inappropriate, I”ll change them

you are right, why is the p tag here?

File: styles/index.css [hunk 1]
```diff
@@ -114,3 +123,9 @@ line-height: 155%;
 display: flex;
 align-items: center;
 }
+.quote {
+  line-height: 1;
+  letter-spacing: -2px;
+  color: gray;
+  font-size: 30px;
+}
```
Comment (akazyryna) [inline styles/index.css]: add empty string below

**Review by akazyryna — COMMENTED** [review]
let's go in order, immediately add styles for tags (body, land etc), and then for classes 

and just for example
`p {`
   `property: value; `
    `}`

between each style add empty string at the above and below

File: html/index.html [hunk 2]
```diff
@@ -121,19 +113,19 @@ <h3>Starting with a staircase</h3>
...
-              </p>
+                the new programming vocabulary, one child</p>
             </article>
       </section>
       <section class="second-container">
         <article>
           <div class="imgblock"></div>
-            <img src="../images/calculator.png" height="80%" width="80%" alt="">
+            <img src="../images/calculator.png" height="60%" width="60%" alt="">
              <span> <p><right> A rod arithmetic calculator built after studying the staircase step <br>
```
Comment (akazyryna) [inline html/index.html]: Why `p` tag in `span`?

File: html/index.html [hunk 3]
```diff
@@ -148,33 +140,36 @@ <h3>Starting with a staircase</h3>
...
                 Colour data type to model odd or even 
                 staircases and test their functions to see 
                 what the effect would be on the derived 
                 successor function. A transcript (see 
-                insert) shows a pupil working with the 
-                number names for the rods, measured 
+                insert) shows a pupil working with the </p>
+            </section>
+            <div class="line"></div>
+             <h2 class="quote"><img src="../images/blackdot.png" class="blackdot"><blockquote>
```
Comment (akazyryna) [inline html/index.html]: Could you make normal indents for tags, it's not readable. Each tag on a new line

Comment (akazyryna) [reply html/index.html]: And need to check this whole doc for compliance

File: html/index.html [hunk 4]
```diff
@@ -186,28 +181,29 @@ <h3>Programming</h3>
...
+       <div class="imgblock"></div>
+       <span> <p><right>Transcript of a conversation about program behaviour </right> </p> </span>
+      <section><img src="../images/dialog.png" height="80%" width="80%" alt=""></section>
+        <div class="line"></div>
+             <article class="block">
+              <div class='imgblock'>
+                <img src="../images/blacksquare_89177.png" >
+             <div> 
+                <p class="caption">WHERE TO GO NEXT</p>
+                <ul>Many Cuisenaire and Gattegno resources can be found online:
```
Comment (akazyryna) [inline html/index.html]: why this text in `ul` tag?

File: html/index.html [hunk 5]
```diff
@@ -186,28 +181,29 @@ <h3>Programming</h3>
...
-                <p><li>The common sense of teaching mathematics: 
-                <a href="../html/index.html">helloworld.cc/2p9dyKO</a></li></p>
-                <p><li>Gattegno Mathematics, Book 1: 
-                <a href="../html/index.html">helloworld.cc/2qz7xZe</a>.</li></p>
-                <p><li>Virtual Rods: 
-                <a href="../html/index.html">helloworld.cc/2pI8ctc</a>.</li></p>
-                <p><li>Sample code for these exercises: 
-                <a href="../html/index.html">helloworld.cc/2pFyx9j</a>.</li></p>
-              </ul></span>
+                <a href="../html/index.html">helloworld.cc/2pYFgOw</a>.</li>
```
Comment (akazyryna) [inline html/index.html]: what is the dot between `</a>` and `</li>`?

Comment (akazyryna) [reply html/index.html]: are you sure that this path (in href) will redirect user to the correct link that is specified in the pdf maket ?
this applies to all of the following links in this block

File: html/index.html [hunk 6]
```diff
@@ -240,10 +236,9 @@ <h3>Calculating</h3>
...
-              </p>
-            </article>
-               </section>
-   </main>
+               may lead us to next!</p>
+          </article>
+      </section>
+    </main>
   </body>
 </html>
```
Comment (akazyryna) [inline html/index.html]: add empty string below

File: styles/index.css [hunk 7]
```diff
@@ -1,5 +1,6 @@
 
 html {
+  line-height: 1.5 ;
```
Comment (akazyryna) [inline styles/index.css]: what is the empty space before the semicolon?

File: styles/index.css [hunk 8]
```diff
@@ -31,9 +32,16 @@ span {
   border-bottom: 4px dotted rgb(136, 137, 142);
 }
 .some-text {
+  line-height: 1;
   font-size: 70px;
   margin: 140px 100px 0 100px;
```
Comment (akazyryna) [inline styles/index.css]: i think you can use like this `margin: 140px 100px 0`

File: styles/index.css [hunk 9]
```diff
@@ -31,9 +32,16 @@ span {
   border-bottom: 4px dotted rgb(136, 137, 142);
 }
 .some-text {
```
Comment (akazyryna) [inline styles/index.css:39]: add empty space above

File: styles/index.css [hunk 10]
```diff
@@ -31,9 +32,16 @@ span {
...
+  line-height: 1;
   font-size: 70px;
   margin: 140px 100px 0 100px;
-  color: purple;
+  color: rgb(50, 49, 50);
+}
```
Comment (akazyryna) [inline styles/index.css:45]: empty space below

File: styles/index.css [hunk 11]
```diff
@@ -31,9 +32,16 @@ span {
...
+h2 {
+
```
Comment (akazyryna) [inline styles/index.css]: delete this empty space

File: styles/index.css [hunk 12]
```diff
@@ -31,9 +32,16 @@ span {
...
   font-size: 70px;
   margin: 140px 100px 0 100px;
-  color: purple;
+  color: rgb(50, 49, 50);
+}
+h2 {
+ 
+  font-weight: lighter;
+  letter-spacing: -1px;
+
```
Comment (akazyryna) [inline styles/index.css]: and this

File: styles/index.css [hunk 13]
```diff
@@ -92,25 +101,50 @@ ul {list-style: none}
...
-display: flex;
-align-items: center;
+.quote {
+  line-height: 1;
+  letter-spacing: -2px;
+  color: gray;
+  font-size: 33px;
+  padding-top: 20px;
+}
+blockquote::before {
```
Comment (akazyryna) [inline styles/index.css:156]: add empty space above

File: styles/index.css [hunk 14]
```diff
@@ -92,25 +101,50 @@ ul {list-style: none}
...
+  content: '\201c';
+  color: #fff;
+  background-color: rgb(13, 13, 13);
+  font-weight: 100%;
+}
```
Comment (akazyryna) [inline styles/index.css:161]: add empty space below

and check whole doc to use this approach

**Comment by KarynaKovhan** [comment]
Done

пн, 9 трав. 2022 р. о 14:48 Alina Kazyryna ***@***.***> пише:

File: html/index.html [hunk 15]
```diff
@@ -10,240 +10,171 @@
...
+       <span>
+        <p>
+          <right>Transcript of a conversation about program behaviour</right>
+        </p>
+      </span>
+      <section>
+        <img src="../images/dialog.png" height="100%" width="100%" alt="">
+      </section>
+        <div class="line"></div>
+             <article class="block">
```
Comment (akazyryna) [inline html/index.html]: as i see, you didnt close this tag

Comment (akazyryna) [reply html/index.html]: and also i dont see a beautiful code structure

Comment (akazyryna) [reply html/index.html]: make sure that all tags are closed, pls

File: styles/index.css [hunk 16]
```diff
@@ -10,37 +10,62 @@ body {
...
 }
+
 .container {
   column-count: 3;
   column-gap: 20px;
   column-rule: 4px dotted rgb(136, 137, 142);
+  margin-right: 200px;
+  margin-left: 200px;
+  margin-top: 5px;
+  margin-bottom: 5px;
```
Comment (akazyryna) [inline styles/index.css]: you can use this `margin: 5px 200px`

File: styles/index.css [hunk 17]
```diff
@@ -10,37 +10,62 @@ body {
...
 }
+
 .second-container {
   margin-top: 20px;
+  margin-left: 200px;
+  margin-right: 200px;
```
Comment (akazyryna) [inline styles/index.css]: also here `margin: 20px 200px 0`

File: styles/index.css [hunk 18]
```diff
@@ -10,37 +10,62 @@ body {
...
+  font-size: 75px;
+  margin: 140px 240px 0px ;
+  color: rgb(50, 49, 50);
 }
+
+h2 {
+  font-weight: lighter;
+  letter-spacing: -1px;
+  margin-right: 200px;
+  margin-left: 200px;
```
Comment (akazyryna) [inline styles/index.css]: here `margin: 0 200px`

File: styles/index.css [hunk 19]
```diff
@@ -10,37 +10,62 @@ body {
...
+}
+
 .title  {
   overflow: hidden;
   text-align: center;
   font-size: 10px;
   margin-bottom: 10px;
+  margin-left: 200px;
+  margin-right: 200px;
```
Comment (akazyryna) [inline styles/index.css]: here `margin: 0 200px 10px`

File: styles/index.css [hunk 20]
```diff
@@ -50,67 +75,128 @@ span {
...
+.quote {
+  line-height: 1;
+  letter-spacing: -2px;
+  color: gray;
+  font-size: 33px;
+  padding-top: 0px;
+  margin-left: -44px;
+  margin-right: -14px;
+  margin: -45px;
+  margin-bottom: -25px;
```
Comment (akazyryna) [inline styles/index.css]: could you use only margin without left right bottom?

**Comment by KarynaKovhan** [comment]
Done.

ср, 11 трав. 2022 р. о 16:37 Alina Kazyryna ***@***.***> пише:

**Pull request merged**